Gemfile

```diff
@@ -54,6 +54,7 @@ gem 'sidekiq'
 # For home page pagination
 gem 'will_paginate', '~> 3.1.1'
 gem 'will_paginate-bootstrap'
+gem 'pg_search'
```

Comment:
put this inside postgres block in Gemfile

Reply:
fixed